### ISSUE TITLE
Ability to configure / change the loading state in the tab: "Updating..."

### ISSUE THREAD

**Issue by chriddyp** [plotly/dash#732]
Right now, `dash-renderer` updates the page tab whenever there is a request in flight.

For fast updating callbacks, e.g. `dcc.Interval`, this becomes pretty annoying: https://community.plot.ly/t/how-to-avoid-updating-html-browser-title-when-using-interval-event/15379

This is one of the few "hardcoded" features in Dash that we should make configurable:
1. We could make this a `config` option to turn `on`/ `off` or change the text
2. We could remove it from `dash-renderer` and attempt to roll it in to loading states. However, loading states are currently configurable only _per-component_, not across the entire app.

Seems like (1) would be the way to go right now. It could be something like 
```
update_page_title_during_requests = true / false / 'Loading ...'
```

**Comment by fl4p** [comment]
It would also be nice to simply have a CSS class added while loading is happening, i.e. `<body class="loading">`

**Comment by stlehmann** [comment]
This would be a great improvement especially when it comes to displaying live data with small intervals (like 100ms). I would like to offer my help implementing this. Maybe someone of the core developers could give a hint where to start. I still couldn' t figure out where in the code the title is changed. I really like the idea of @chriddyp to add an attribute in the config:

> Seems like (1) would be the way to go right now. It could be something like
> 
> update_page_title_during_requests = true / false / 'Loading ...'

**Comment by stlehmann** [comment]
The title is set in DocumentTitle.react.js:

```
import {connect} from 'react-redux';
import {Component} from 'react';
import PropTypes from 'prop-types';

class DocumentTitle extends Component {
    constructor(props) {
        super(props);
        this.state = {
            initialTitle: document.title,
        };
    }

    UNSAFE_componentWillReceiveProps(props) {
        if (props.pendingCallbacks.length) {
            document.title = 'Updating...';
        } else {
            document.title = this.state.initialTitle;
        }
    }

    shouldComponentUpdate() {
        return false;
    }

    render() {
        return null;
    }
}
```

**Comment by chriddyp** [comment]
This feature is now available in `dash==1.14.0` (thanks to community member @stlehmann !) and is documented in http://dash.plotly.com/external-resources. Look for the `title=` and `update_title=`. See the "[Customizing Dash's Document or Browser Tab Title]( http://dash.plotly.com/external-resources)", " [Update the Document Title Dynamically based off of the URL or Tab]( http://dash.plotly.com/external-resources)", and "[Customizing or Removing Dash's "Updating..." Message](http://dash.plotly.com/external-resources)" sections.
